sbol_utilities/sbol3_genbank_conversion.py

```diff
@@ -71,6 +75,34 @@ def build_feature_qualifiers_extension(*, identity, type_uri) -> GenBank_SBOL3_C
             obj.clear_property(sbol3.SBOL_TYPE)
             return obj
 
+        # def build_range_extension(*, identity, type_uri) -> GenBank_SBOL3_Converter.Range_GenBank_Extension:
```

Comment:
- [ ] Please delete this commented code before merging.

Reply:
Done